README.md

```diff
@@ -3,15 +3,6 @@
 This is a central place for deploying SDR applications (and sul_pub).  This allows all applications
 to be deployed at once.
 
-## Setup
```

Comment:
Could someone on @sul-dlss/infrastructure-team pls verify that an explicit `gem install bundler` is not require to run/use sdr-deploy? If so, this removal is OK. If not, we need to add `gem install bundler` to the README.

Reply:
i checked out this branch, ran `./deploy.rb qa --checkssh`, and got errors like:

```
li-dl-7068-0345:sdr-deploy suntzu$ ./deploy.rb qa --checkssh
Updated 0 paths from the index
Installing gems for tmp/repos/sul-dlss/argo
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:105: warning: already initialized constant Bundler::FileUtils::VERSION
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:105: warning: previous definition of VERSION was here
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1284: warning: already initialized constant Bundler::FileUtils::Entry_::S_IF_DOOR
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1284: warning: previous definition of S_IF_DOOR was here
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1568: warning: already initialized constant Bundler::FileUtils::Entry_::DIRECTORY_TERM
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1568: warning: previous definition of DIRECTORY_TERM was here
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1626: warning: already initialized constant Bundler::FileUtils::OPT_TABLE
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1626: warning: previous definition of OPT_TABLE was here
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1685: warning: already initialized constant Bundler::FileUtils::LOW_METHODS
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1685: warning: previous definition of LOW_METHODS was here
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1692: warning: already initialized constant Bundler::FileUtils::METHODS
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1692: warning: previous definition of METHODS was here
Traceback (most recent call last):
	9: from /Users/suntzu/.rbenv/versions/2.7.1/bin/bundle:23:in `<main>'
	8: from /Users/suntzu/.rbenv/versions/2.7.1/bin/bundle:23:in `load'
	7: from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/exe/bundle:13:in `<top (required)>'
	6: from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/exe/bundle:13:in `require_relative'
	5: from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler.rb:7:in `<top (required)>'
	4: from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler.rb:7:in `require_relative'
	3: from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/errors.rb:3:in `<top (required)>'
	2: from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/errors.rb:20:in `<module:Bundler>'
	1: from /Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/errors.rb:20:in `<class:GemfileError>'
/Users/suntzu/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/lib/bundler/errors.rb:9:in `status_code': Trying to register Bundler::GemfileError for status code 4 but Bundler::GemfileError is already registered (ArgumentError)
```

but this happened both before and after i called `gem install bundler`.  also, vague recollectons of doing `gem install bundler` anyway a few days ago, when moving to ruby 2.7.1 on a different project (sorry, can't remember exactly when, or which project).

Reply:
same command ran fine after switching back to `master`

Reply:
@jmartin-sul mind trying again?

Reply:
so... the change now runs fine for me, but the output is also a little less usable, because all the bundler output for each project lands in the output to the user too, e.g.:
![Screen Shot 2020-07-16 at 6 45 46 PM](https://user-images.githubusercontent.com/7741604/87739630-27cca900-c795-11ea-9b8e-627d924b55ef.png)

vs

![Screen Shot 2020-07-16 at 6 51 17 PM](https://user-images.githubusercontent.com/7741604/87739707-5b0f3800-c795-11ea-9dda-89d730491ef4.png)

do with that as you will -- i wouldn't stop the PR over it, though i personally have a slight preference for not mixing the repeated bundle output in the `--checkssh` output (i care about it less for the actual deployment, since that output is already so voluminous).

Reply:
@jmartin-sul ah. I can reduce the amount of bundler crud by replacing `bundle install` with `bundle install --quiet`. Good change to make?

Reply:
oh nice, yeah, i'd appreciate that if you wouldn't mind